### DISCUSSION TITLE
Is it possible to make 1000 fake pods running on a node in K3s using virtual kubelet and silly drivers

### DISCUSSION THREAD

**Discussion by vamsitalluru** [k3s-io/k3s#5320] [Q&A]
Hi,

I have K3s installed along with virtual kubelet and silly provider. I have created about 1000 fake pods on a node. out of which about 109 pods are in running state. 
Is it possible to have all the 1000 pods running, is there any way to do?

Links used to install virtual kubelet and silly provider
Virtual Kubelet-> https://github.com/virtual-kubelet/virtual-kubelet
Silly provider-> https://github.com/lizrice/virtual-kubelet/tree/silly/providers/silly.

Any insights would be of great help.

Thank you in advance
VT

**Comment by brandond** [comment]
I have never used either of these projects, but it sounds like you might need to raise max-pods on the virtual kubelet? Max-pods usually defaults to 110.
